shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -451,15 +451,17 @@ - (void)setSemanticsEnabled:(BOOL)enabled {
     return;
   }
   _semanticsEnabled = enabled;
+  // We need to remove the accessibility children from flutter view
+  // before we reset the bridge.
+  if (!_semanticsEnabled) {
+    self.viewController.flutterView.accessibilityChildren = nil;
```

Comment:
This will cause the view to load if it's not already loaded, which is probably undesirable. This should probably be something like

```
if (!_semanticsEnabled && self.viewController.viewLoaded) {
```

To avoid that.